src/applications/vaos/new-appointment/redux/actions.js

```diff
@@ -717,11 +720,15 @@ export function checkCommunityCareEligibility() {
 
       // Reroute to VA facility page if none of the user's registered systems support community care.
       if (ccEnabledSystems.length) {
-        const response = await getCommunityCare(getCCEType(state));
-
+        let response = null;
+        if (useV2) {
+          response = await getCommunityCareV2(getCCEType(state));
+        } else {
+          response = await getCommunityCare(getCCEType(state));
+        }
         dispatch({
           type: FORM_UPDATE_CC_ELIGIBILITY,
-          isEligible: response.eligible,
+          isEligible: response.eligible || response.length > 0,
```

Comment:
If latest v2 cc eligibility returns a boolean, why do we need a check for array length?

Reply:
Good catch @jenniemc -  I meant to take that out